Makefile

```diff
@@ -0,0 +1,20 @@
+
+build_all: klaw_core cluster_api
+
+klaw_core:
+	cd core && mvn clean verify
+
+cluster_api:
+	cd cluster-api && mvn clean verify
+
+edit-core-config:
+	${EDITOR} core/target/classes/application.properties
+
+edit-cluster-api-config:
+	${EDITOR} cluster-api/target/classes/application.properties
+
+run-core:
+	java -jar core/target/klaw-1.1.0.jar
```

Comment:
Is there a way to use some constant/var for a version number to minimize number of lines to change when new release happening?